python/ray/rllib/models/modelv2.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from collections import namedtuple
+from ray.rllib.models.model import restore_original_dimensions
+from ray.rllib.utils import try_import_tf
+
+tf = try_import_tf()
```

Comment:
this and the `try_import_tf` doesn't seem to be needed in this file (and it would make sense for the file to be framework agnostic)